evm-contracts/src/v0.7/dev/OperatorGenerator.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity ^0.7.0;
+
+import "./Operator.sol";
+
+/**
+ * @title Operator Generator
+ * @notice Generates Operator contracts for node operators
+ */
+contract OperatorGenerator {
+
+    address public link;
+
+    event OperatorCreated(address indexed operator, address indexed owner);
+
+    /**
+     * @param linkAddress address
+     */
+    constructor(address linkAddress) public {
+        link = linkAddress;
+    }
+
+    /**
+     * @notice Create a new Operator contract with the msg.sender as owner
+     * @return operatorAddress
+     */
+    function createOperator() external returns (address operatorAddress){
+        Operator operator = new Operator(link, msg.sender);
+        operatorAddress = address(operator);
+        emit OperatorCreated(operatorAddress, msg.sender);
+    }
+}
```

Comment:
Do we want to add anything else to this function, like calling `setFulfillmentPermission`?

Reply:
No, I think we should do as little configuration as possible. We really won't even know what address permissions to set anyway.